package.json

```diff
@@ -17,7 +17,7 @@
     "react-dom": "^18.2.0",
     "react-redux": "^8.0.5",
     "react-router": "^6.7.0",
-    "react-router-dom": "^6.7.0",
+    "react-router-dom": "^6.9.0",
```

Comment:
upgrading not required and also remove package.json and package-lock.json from your PR